### PR TITLE
fix(frontend-projects): replace video with image thumbnail

### LINKED ISSUE #12753: Create a placeholder project view until multi-tab editor is ready
Currently @benmcmahon100 is working on updating our UI and creating a multi-file editor. This will enable campers to build non-backend projects right on freeCodeCamp instead of having to use CodePen.

We are eliminating videos from our projects views and replacing them with static images. I've added imgur links already as the challenge seed for each of the non-backend projects: 
```
      "challengeSeed": [
        "G6iPE6W"
      ],
```

This should translate to http://i.imgur.com/G6iPE6W.png as the image source.

Currently the project views look pretty terrible:

### LINKED ISSUE #14114: Build a Tribute Page - no user stories shown
Challenge [build-a-tribute-page](http://beta.freecodecamp.com/en/challenges/applied-responsive-web-design-projects/build-a-tribute-page) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36</code>.

The text in the challenge says "Fulfill the user stories by getting all of the tests to pass. Use whichever libraries you need. Give it your own personal style."

However there are no user stories listed? Should these not be shown on the page?

### PR DESCRIPTION
**Pull request by raisedadead**
Closes #12753
Closes #14114

### CODE REVIEW

**Comment by raisedadead** [comment]
This is my first "REAL" take at fixing something in react I think, please let me know in case I did something terribly wrong. 

Earlier, I had fixed things by eyeballing lol.

File: common/app/routes/challenges/views/project/Project.jsx [hunk 1]
```diff
@@ -18,34 +17,35 @@ const mapStateToProps = createSelector(
...
         description,
-        challengeSeed: [ videoId = '' ] = []
+        challengeSeed
       } = {},
       title
     }
   ) => ({
     id,
-    videoId,
+    challengeSeed,
```
Comment (BerkeleyTrue) [inline common/app/routes/challenges/views/project/Project.jsx]: We should rename this var to something more descriptive like `img`.

**Comment by raisedadead** [comment]
@BerkeleyTrue I have updated the property and have also changed the same in the seed.

**Comment by BerkeleyTrue** [comment]
@raisedadead I see you also changed the property in the challenge seed. Not my intention but I think that is a good way to go. Since there will only be one image let's change the `img` key in the seed to `image` and the value to a string corresponding to image path.

We will also need to add it to the challenge.json schema file. We should also add a description there stating that it is used with project challenges

**Comment by BerkeleyTrue** [comment]
Code LGTM. I'll pull it down to test

**Comment by BerkeleyTrue** [comment]
@raisedadead Beautiful work!

**Comment by raisedadead** [comment]
@BerkeleyTrue thanks for the QA.

**Comment by QuincyLarson** [comment]
@raisedadead Nicely done!

**Pull request merged**